.ci/config/matrices.yaml

```diff
@@ -38,6 +39,7 @@ daily-stages:
   - 'mariadb_106'
   - 'mysql_80'
   - 'oracle_19'
+  - 'oracle_23'
```

Comment:
💭 Please note that we usually keep only one version (latest) per database for the daily stages. See line 35-36. Maybe we can keep `oracle_19` for 7.22 and switch to `oracle_23` in 7.23 so the old engine suite is not executed. But that mean `sql-script` won't run with Oracle 23.

Reply:
Or we keep both and I try to skip the old-engine stage for oracle 23.

Reply:
I created https://github.com/camunda/camunda-bpm-platform/issues/4556